aiortc/rtcrtpsender.py

```diff
@@ -192,7 +193,12 @@ def _send_keyframe(self):
         loop = asyncio.get_event_loop()
 
         encoder = get_encoder(codec)
-        packet = RtpPacket(payload_type=codec.payloadType)
+        # init randomly as rfc3550 section-5.1
```

Comment:
These changes are all legitimate but unrelated to H.264, a separate PR would have been better.

I've integrate the fixes on `master` so you can remove them from this PR.